save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt

```diff
@@ -70,19 +70,28 @@ internal class OrganizationController(
 ) {
     private val webClientToPreprocessor = WebClient.create(config.preprocessorUrl)
 
+    /**
+     * @param includeDeleted whether deleted organizations should be included in
+     *   the response.
+     *   The default is `true`.
+     * @return the list of organizations.
+     */
     @GetMapping("/all")
     @PreAuthorize("permitAll()")
     @Operation(
         method = "GET",
         summary = "Get all organizations",
         description = "Get organizations",
     )
-    @Parameters(
-        Parameter(name = "organizationName", `in` = ParameterIn.PATH, description = "name of an organization", required = true),
-    )
     @ApiResponse(responseCode = "200", description = "Successfully fetched all registered organizations")
-    fun getAllOrganizations() = Mono.fromCallable {
-        organizationService.findAll()
+    fun getAllOrganizations(
+        @RequestParam(required = false, defaultValue = "true") includeDeleted: Boolean
```

Comment:
right now all is `created` + `deleted`, but as I got from stand-up. we are going to add `BAN` status.
maybe rename this property to somethin like `onlyActive`?

Reply:
...and invert the flag meaning.

Thanks, will do.